crypto/fipsmodule/FIPS.md

```diff
@@ -43,7 +43,7 @@ The utility in `util/fipstools/break-hash.go` can be used to corrupt the FIPS mo
 
 ## RNG design
 
-FIPS 140-2 requires that one of its PRNGs be used (which they call DRBGs). In BoringCrypto, we use CTR-DRBG with AES-256 exclusively and `RAND_bytes` (the primary interface for the rest of the system to get random data) takes its output from there.
+FIPS 140-2 requires that one of its PRNGs be used (which they call DRBGs). In BoringCrypto, we use CTR-DRBG with AES-128 or AES-256. `RAND_bytes` (the primary interface for the rest of the system to get random data) takes its output from there using the AES-256 mode of CTR-DRBG.
```

Comment:
This means that AES-128 can be used only internally? We don't provide public API for choosing between AES-128 and AES-256?

Reply:
For now, yes. Check out CryptoAlg-730.